### ISSUE TITLE
Game crashes randomly

### ISSUE THREAD

**Issue by ghost** [jellysquid3/Sodium#1089]
### Version information

sodium-fabric-mc1.18.1-rc1-0.4.0-alpha5+build.835

### Reproduction Steps

I don't know. it just randomly crashes with no consistency.

### Crash Report file

---- Minecraft Crash Report ----
// Oh - I know what I did wrong!

Time: 30.12.21 14:07
Description: Unexpected error

java.lang.IllegalStateException: Tried to mark light data as ready for chunk [-15, -20] but it hasn't been loaded yet
	at me.jellysquid.mods.sodium.client.render.chunk.ChunkTracker.onLightDataAdded(ChunkTracker.java:92)
	at me.jellysquid.mods.sodium.client.render.SodiumWorldRenderer.onChunkLightAdded(SodiumWorldRenderer.java:304)
	at net.minecraft.class_638.handler$bhh000$postLightUpdate(class_638.java:2578)
	at net.minecraft.class_638.method_39849(class_638.java:530)
	at net.minecraft.class_634.method_38543(class_634.java:2198)
	at net.minecraft.class_634.method_38545(class_634.java:2183)
	at net.minecraft.class_638.method_38534(class_638.java:142)
	at net.minecraft.class_761.method_22710(class_761.java:1208)
	at net.minecraft.class_757.method_3188(class_757.java:1031)
	at net.minecraft.class_757.method_3192(class_757.java:811)
	at net.minecraft.class_310.method_1523(class_310.java:1117)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at me.jellysquid.mods.sodium.client.render.chunk.ChunkTracker.onLightDataAdded(ChunkTracker.java:92)
	at me.jellysquid.mods.sodium.client.render.SodiumWorldRenderer.onChunkLightAdded(SodiumWorldRenderer.java:304)
	at net.minecraft.class_638.handler$bhh000$postLightUpdate(class_638.java:2578)
	at net.minecraft.class_638.method_39849(class_638.java:530)
	at net.minecraft.class_634.method_38543(class_634.java:2198)
	at net.minecraft.class_634.method_38545(class_634.java:2183)
	at net.minecraft.class_638.method_38534(class_638.java:142)
	at net.minecraft.class_761.method_22710(class_761.java:1208)
	at net.minecraft.class_757.method_3188(class_757.java:1031)

-- Affected level --
Details:
	All players: 1 total; [class_746['Gollug'/336330, l='ClientLevel', x=-255.64, y=19.42, z=-431.61]]
	Chunk stats: 1024, 307
	Level dimension: minecraft:overworld
	Level spawn location: World: (-107,5,-389), Section: (at 5,5,11 in -7,0,-25; chunk contains blocks -112,0,-400 to -97,255,-385), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 1165 game time, 1000 day time
	Server brand: Paper
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:408)
	at net.minecraft.class_310.method_1587(class_310.java:2402)
	at net.minecraft.class_310.method_1514(class_310.java:757)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 2
	Reload reason: manual
	Finished: Yes
	Packs: Default, Fabric Mods, VanillaTweaks 1.18.zip

-- System Details --
Details:
	Minecraft Version: 1.18.1
	Minecraft Version ID: 1.18.1
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 2442269104 bytes (2329 MiB) / 4731174912 bytes (4512 MiB) up to 8589934592 bytes (8192 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i7-5500U CPU @ 2.40GHz
	Identifier: Intel64 Family 6 Model 61 Stepping 4
	Microarchitecture: Broadwell (Client)
	Frequency (GHz): 2,39
	Number of physical packages: 1
	Number of physical CPUs: 2
	Number of logical CPUs: 4
	Graphics card #0 name: NVIDIA GeForce 840M
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 2048,00
	Graphics card #0 deviceId: 0x1341
	Graphics card #0 versionInfo: DriverVersion=30.0.14.7111
	Memory slot #0 capacity (MB): 8192,00
	Memory slot #0 clockSpeed (GHz): 1,60
	Memory slot #0 type: Unknown
	Virtual memory max (MB): 16205,13
	Virtual memory used (MB): 7605,24
	Swap memory total (MB): 8101,64
	Swap memory used (MB): 0,00
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		bettercontrols: Better Controls 1.2.3
		blue_endless_jankson: jankson 1.2.1
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-config2: Cloth Config v4 5.0.34
		cmdkeybind: Command Macros 1.5.3-1.18
		custom-crosshair-mod: Custom Crosshair Mod 1.3.8-fabric
		fabric: Fabric API 0.45.0+1.18
		fabric-api-base: Fabric API Base 0.4.1+b4f4f6cd65
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.5.0+17be577f65
		fabric-biome-api-v1: Fabric Biome API (v1) 6.0.1+ded849a965
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.9+3ac43d9565
		fabric-command-api-v1: Fabric Command API (v1) 1.1.6+3ac43d9565
		fabric-commands-v0: Fabric Commands (v0) 0.2.5+b4f4f6cd65
		fabric-containers-v0: Fabric Containers (v0) 0.1.18+d154e2c665
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.5+6f53a73d65
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.8+3ac43d9565
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.7+43d2957165
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.5+6b21378a65
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.16+bfa23f1765
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.6+b4f4f6cd65
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.10+3ac43d9565
		fabric-item-api-v1: Fabric Item API (v1) 1.3.0+691a79b565
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.3+3ac43d9565
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.8+c8aba2f365
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.6+b4f4f6cd65
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.10+c15ca33565
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.8+3ac43d9565
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.3+3ac43d9565
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.7+b4f4f6cd65
		fabric-models-v0: Fabric Models (v0) 0.3.3+3ac43d9565
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.18+3ac43d9565
		fabric-networking-v0: Fabric Networking (v0) 0.3.5+b4f4f6cd65
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.0+3b82842e65
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.8+3ac43d9565
		fabric-particles-v1: Fabric Particles (v1) 0.2.9+526dc1ac65
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.8.6+533be9ba65
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.9+3ac43d9565
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.12+3ac43d9565
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.7+b4f4f6cd65
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.4+7242e9d765
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.18+3ac43d9565
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.9+b4f4f6cd65
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.3+6b21378a65
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.11+3ac43d9565
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.7+3ac43d9565
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.11+3ac43d9565
		fabric-structure-api-v1: Fabric Structure API (v1) 2.0.8+295197a765
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.5+3ac43d9565
		fabric-textures-v0: Fabric Textures (v0) 1.0.9+3ac43d9565
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.4+7de09f5565
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.6+b4f4f6cd65
		fabricloader: Fabric Loader 0.12.12
		hydrogen: Hydrogen 0.3-SNAPSHOT
		java: OpenJDK 64-Bit Server VM 17
		kyrptconfig: Kyrpt Config 1.2.6-1.18
		libzoomer: LibZoomer 0.3.0+1.17.1
		lithium: Lithium 0.7.7-SNAPSHOT
		minecraft: Minecraft 1.18.1
		modmenu: Mod Menu v3.0.1+unknown.fd8079b
		mousetweaks: Mouse Tweaks 2.20
		multiconnect: MultiConnect 1.5.7
		multiconnect-api: MultiConnect API 1.5.7
		okzoomer: Ok Zoomer 5.0.0-beta.3+1.17.1
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_joml_joml: joml 1.10.2
		org_quiltmc_quilt-json5: quilt-json5 1.0.0
		perspectivemod: Perspective Mod Redux 0.0.5_01
		phosphor: Phosphor 0.8.1
		sodium: Sodium 0.4.0-alpha5+build.835
		sodium-extra: Sodium Extra 0.3.7
		spruceui: SpruceUI 3.3.0+1.17
		torohealth: ToroHealth Damage Indicators 1.18-fabric-2
		xaerominimap: Xaero's Minimap 21.23.0
		xaeroworldmap: Xaero's World Map 1.19.0
	Launched Version: fabric-loader-0.12.12-1.18.1
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce 840M/PCIe/SSE2 GL version 3.2.0 NVIDIA 353.82, NVIDIA Corporation
	Window size: 1366x768
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: vanilla, Fabric Mods, file/VanillaTweaks 1.18.zip
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i7-5500U CPU @ 2.40GHz

**Comment by look-a-furry** [comment]
Same issue here, but mine can be reproduced, it only happens when I load a world and I spawn in the nether, when going out of the portal, the games tries to load the overworld but crashes as soon as the first chunk is loaded.
This was not happening in 1.18 but I joined a paper based server that is on 1.18.1 and it crashes the first time I try to load the overworld. It happens both on singleplayer and multiplayer worlds.

**Comment by Derkades** [comment]
This happens to us too on a paper 1.18-1.18.1 server, occasionally when going through portals

**Comment by vplanida** [comment]
I'm not sure if the problem is related, but my private server is crashing with 3-4 people on the server. 
More than once we have crashed while one person died in the nether and tried to get back to get his stuff back from death chest.
[Spark registers tick rise on net.minecraft.server.world.ServerChunkManager (1640ms)](https://spark.lucko.me/0YOdQxjjLO)

I am hugely sorry, if my problem is not related to Sodium. I'm really lost when it's coming to fabric's error logging.
Files:
[Full crash-2022-01-02_21.27.24-server.txt](https://github.com/CaffeineMC/sodium-fabric/files/7799918/crash-2022-01-02_21.27.24-server.txt)

---- Minecraft Crash Report ----
// Oh - I know what I did wrong!

Time: 1/2/22, 9:27 PM
Description: Watching Server

java.lang.Error: Watchdog
	at net.minecraft.class_3196.method_15486(class_3196.java:45)
	at net.minecraft.class_3554.method_15482(class_3554.java:159)
	at net.minecraft.class_3554.method_15484(class_3554.java:191)
	at net.minecraft.class_3196.method_15487(class_3196.java:27)
	at net.minecraft.class_3554.method_15492(class_3554.java:221)
	at net.minecraft.class_3204$class_3205.method_14057(class_3204.java:298)
	at net.minecraft.class_3204$class_3948.method_14057(class_3204.java:364)
	at net.minecraft.class_3204.method_15892(class_3204.java:106)
	at net.minecraft.class_3215.method_16155(class_3215.java:284)
	at net.minecraft.class_3215.method_12127(class_3215.java:322)
	at net.minecraft.class_3218.method_18765(class_3218.java:372)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:891)
	at net.minecraft.class_3176.method_3813(class_3176.java:335)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:831)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:697)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)
	at net.minecraft.server.MinecraftServer$$Lambda$7558/0x0000000801bf9b70.run(Unknown Source)
	at java.base@16.0.1/java.lang.Thread.run(Thread.java:831)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server Watchdog
Stacktrace:
	at net.minecraft.class_3196.method_15486(class_3196.java:45)
	at net.minecraft.class_3554.method_15482(class_3554.java:159)
	at net.minecraft.class_3554.method_15484(class_3554.java:191)
	at net.minecraft.class_3196.method_15487(class_3196.java:27)
	at net.minecraft.class_3554.method_15492(class_3554.java:221)
	at net.minecraft.class_3204$class_3205.method_14057(class_3204.java:298)
	at net.minecraft.class_3204$class_3948.method_14057(class_3204.java:364)
	at net.minecraft.class_3204.method_15892(class_3204.java:106)
	at net.minecraft.class_3215.method_16155(class_3215.java:284)
	at net.minecraft.class_3215.method_12127(class_3215.java:322)
	at net.minecraft.class_3218.method_18765(class_3218.java:372)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:891)
	at net.minecraft.class_3176.method_3813(class_3176.java:335)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:831)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:697)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)

-- Thread Dump --
Details:
	Threads: "Reference Handler" daemon prio=10 Id=2 RUNNABLE
	at java.base@16.0.1/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
	at java.base@16.0.1/java.lang.ref.Reference.processPendingReferences(Reference.java:243)
	at java.base@16.0.1/java.lang.ref.Reference$ReferenceHandler.run(Reference.java:215)


"Finalizer" daemon prio=8 Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@4561b0a6
	at java.base@16.0.1/java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@4561b0a6
	at java.base@16.0.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
	at java.base@16.0.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
	at java.base@16.0.1/java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:171)

-- System Details --
Details:
	Minecraft Version: 1.17.1
	Minecraft Version ID: 1.17.1
	Operating System: Linux (amd64) version 5.4.0-91-generic
	Java Version: 16.0.1, Private Build
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Private Build
	Memory: 4414472 bytes (4 MiB) / 2147483648 bytes (2048 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Pentium(R) CPU G4600 @ 3.60GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 9
	Microarchitecture: Kaby Lake
	Frequency (GHz): 3.60
	Number of physical packages: 1
	Number of physical CPUs: 2
	Number of logical CPUs: 4
	JVM Flags: 1 total; -Xmx2G
	Fabric Mods: 
		adorn: Adorn 2.3.1+1.17.1
		adventurez: AdventureZ 1.4.3
		appleskin: AppleSkin mc1.17.1-2.2.0
		architects_palette: Architect's Palette Fabric 1.3
		architectury: Architectury 2.8.53
		artifacts: Artifacts 5.0.2+fabric
		attributefix: AttributeFix 11.0.2
		awesomedungeon: Awesome Dungeon 1.1.3
		balm-fabric: Balm 1.9.0+0
		bambootweaks: Bamboo Tweaks 3.0.2
		bannerpp: Banner++ 2.0.4+mc.1.17-rc1
		bclib: BCLib 0.5.6
		betterdungeons: YUNG's Better Dungeons 1.17-1.0.2
		betterend: Better End 0.12.5
		bettermineshafts: YUNG's Better Mineshafts 1.17-1.0.1
		betternether: Better Nether 5.3.14
		betterstrongholds: YUNG's Better Strongholds 1.17-1.1.2
		biom4st3r_repackaged_expressions: expj4_repackaged 0.1.0
		blue_endless_jankson: jankson 1.2.1
		bluemap: BlueMap 1.6.3
		botania: Botania 1.17.1-422-FABRIC
		cardinal-components-base: Cardinal Components API (base) 3.1.1
		cardinal-components-block: Cardinal Components API (blocks) 3.0.0
		cardinal-components-entity: Cardinal Components API (entities) 3.1.1
		chipped: Chipped 1.1.1
		cloth-basic-math: cloth-basic-math 0.6.0
		cloth-config2: Cloth Config v5 5.2.47
		clumps: Clumps 7.0.10
		com_electronwill_night-config_core: core 3.6.5
		com_electronwill_night-config_toml: toml 3.6.5
		com_eliotlash_mclib_mclib: mclib 18
		com_eliotlash_molang_molang: molang 18
		com_github_shevek_parallelgzip: parallelgzip af5f5c297e735f3f2df7aa4eb0e19a5810b8aff6
		com_typesafe_config: config 1.4.1
		com_velocitypowered_velocity-native: velocity-native 1.1.0-SNAPSHOT
		compostable-rottenflesh: Compostable Rotten Flesh 1.0.0
		confabricate: confabricate 2.2.0-SNAPSHOT+4.1.1
		convenient-mobgriefing: Convenient mobGriefing 2.0.2
		couplings: Couplings 1.6.0+1.17
		croptopia: Croptopia 1.7.0
		dual_riders: DualRiders 1.0.5
		dynview: Dynamic View Mod 1.17.1-1.5
		easywhitelist: Easy Whitelist Mod 0.2.0
		expandability: ExpandAbility 4.0.2
		experiencebugfix: Experience Bug Fix 17.0.0
		fabric: Fabric API 0.44.0+1.17
		fabric-api-base: Fabric API Base 0.4.0+5847535018
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.4.0+cb9053d118
		fabric-biome-api-v1: Fabric Biome API (v1) 3.2.2+cf39a74318
		fabric-command-api-v1: Fabric Command API (v1) 1.1.4+cf39a74318
		fabric-commands-v0: Fabric Commands (v0) 0.2.3+cf39a74318
		fabric-containers-v0: Fabric Containers (v0) 0.1.13+cf39a74318
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.1+3447790d18
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.6+cf39a74318
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.13+cf39a74318
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.1+377137cc18
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.12+e99fbe1218
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.4+cf39a74318
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.8+cf39a74318
		fabric-item-api-v1: Fabric Item API (v1) 1.3.0+6617390918
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.2+cf39a74318
		fabric-language-kotlin: Fabric Language Kotlin 1.7.0+kotlin.1.6.0
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.6+0392f3a618
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.5+cf39a74318
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.3+cf39a74318
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.6+cf39a74318
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.14+5847535018
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.12+cf39a74318
		fabric-networking-v0: Fabric Networking (v0) 0.3.3+cf39a74318
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.0+f907116918
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.7+cf39a74318
		fabric-particles-v1: Fabric Particles (v1) 0.2.5+cf39a74318
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.13+cf39a74318
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.6+cf39a74318
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.2.1+cf39a74318
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.10+f09604ce18
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.9+cf39a74318
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.15+cf39a74318
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.3+cf39a74318
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.2+cf39a74318
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.3+cf39a74318
		fabricloader: Fabric Loader 0.12.12
		fallingtree: FallingTree 2.14.5
		fastfurnace: FastFurnace 1.17-3.0
		ferritecore: FerriteCore 3.0.3
		fiber: fiber 0.23.0-2
		geckolib3: Geckolib 3.0.30
		go-fish: Go Fish 1.3.1-1.17.1
		graveyard: The Graveyard 2.1
		harvest: Harvest 1.2.12
		helpwanted: Help Wanted 1.1.2
		hologram-api: Hologram API 0.2.1+1.17.1
		inventorysorter: Inventory Sorter 1.8.0-1.17
		io_leangen_geantyref_geantyref: geantyref 1.3.11
		ironchests: Iron Chests for Fabric 1.4.5
		jankson: Jankson 3.0.1+j1.2.0
		java: OpenJDK 64-Bit Server VM 16
		krypton: Krypton 0.1.4
		kyrptconfig: Kytpt Config 1.1.10-1.17
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 4.2.2+1.17.1
		libninepatch: LibNinePatch 1.1.0
		lithium: Lithium 0.7.5
		minecraft: Minecraft 1.17.1
		mo_glass: Mo Glass 1.6.1-MC1.17.1
		moenchant_lib: Mo'Enchantment Library 0.2.0
		moenchantments: Mo'Enchantments 1.18.1
		morebannerfeatures: More Banner Features 1.1.2
		morevillagers-fabric: MoreVillagersFabric 2.1.1-SNAPSHOT
		mostructures: Mo' Structures 1.3.0-pre1-1.17.1
		netherportalfix: NetherPortalFix 8.0.0+0
		no_null_processors: No Null Processors - Fabric 1.0.0+1.17.1
		omega-config: OmegaConfig 1.0.8
		org_apache_commons_commons-compress: commons-compress 1.19
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.6.0
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.6.0
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.6.0
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.6.0
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.2
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.2
		org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.1
		org_spongepowered_configurate-core: configurate-core 4.1.1
		org_spongepowered_configurate-extra-dfu4: configurate-extra-dfu4 4.1.1
		org_spongepowered_configurate-gson: configurate-gson 4.1.1
		org_spongepowered_configurate-hocon: configurate-hocon 4.1.1
		org_tukaani_xz: xz 1.8
		packet_tweaker: Packet Tweaker 0.2.0+1.17.1
		pandamc_concreteplus: Concrete plus 1.9
		particle_emitter_biom4st3r: Particle Emitters LIB 0.1.0
		patchouli: Patchouli 1.17.1-59-FABRIC
		phosphor: Phosphor 0.8.0
		placeholder-api: Placeholder API 1.1.2+1.17.1
		plasmo_voice: Plasmo Voice 1.2.9
		polymer: Polymer 0.1.8+1.17.1
		quarrymod: Industial Quarry Mod 0.5.2
		reach-entity-attributes: Reach Entity Attributes 2.1.1
		reborncore: Reborn Core 5.0.11-beta
		repurposed_structures: Repurposed Structures 2.5.7+1.17.1
		reroll: Reroll 1.3.0-1.17.1
		roughlyenoughitems: Roughly Enough Items 6.3.355
		server_translations_api: Server Translations API 1.4.8+1.17
		sgui: sgui 1.0.0-rc5+1.17.1
		someforgepatchesported: Some Forge Patches Ported 1.0.1
		spark: spark 1.6.3
		step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		stoneholm: Stoneholm 1.3
		team_reborn_energy: Energy 2.0.0-beta1
		techreborn: Tech Reborn 5.0.11-beta
		textile_backup: Textile Backup 2.2.0-1.17
		trinkets: Trinkets 3.0.4
		universal-graves: Universal Graves 1.1.4+1.17.1
		voidz: VoidZ 1.0.5
		waystones: Waystones 2.2.2
		xaerominimapfair: Xaero's Minimap 21.22.3
		yungsapi: YUNG's API 1.17-Fabric-19
	
	Data Packs: vanilla, Fabric Mods
	Is Modded: Definitely; Server brand changed to 'fabric'
	Type: Dedicated Server (map_server.txt)

**Comment by CyGyGames** [comment]
Same thing to me I want to launch Sodium with my Fabeic Mods on a Server but It crashed either when I join or before this needs to be fixed Sodium is a really needed mod for an Mod Server with 60 Mods

**Comment by Millsonhub** [comment]
This happens way too often for me too

**Comment by davigamer987** [comment]
I also have this issue

**Comment by silversquirl** [comment]
Duplicate of #1016
